lib/rules/selector-max-type/__tests__/index.mjs

```diff
@@ -109,25 +106,25 @@ testRule({
 			line: 1,
 			column: 13,
 			endLine: 1,
-			endColumn: 20,
+			endColumn: 21,
 		},
 		{
 			code: '.foo { div {} }',
```

Comment:
question

I suppose `code: 'p { div {} }',` gives `message: messages.expected('p div', 0),`?
Or does it count as 2 separate errors?

Reply:
Currently it would be reported as `div`.
My thinking was that the resolved selector (`p div`) wouldn't exist in source.

So a user who got a warning for `p div` might do a find throughout their codebase for exactly `p div` and wouldn't get any results.

But it can be easily changed so that the resolved selector is reported instead or together with the current value.

I don't have a strong preference here.

Reply:
If you thought about it already, it's fine.